.github/workflows/cmake.yml

```diff
@@ -145,20 +152,38 @@ jobs:
 #    - name: Clear ccache
 #      run: ccache --clear
 
+    - name: Checkout SOPT
+      uses: actions/checkout@v3
+      with:
+        repository: astro-informatics/sopt.git
+        path: sopt
+        ref: development
+
     - name: Build sopt
       run: |
-        conan profile detect
-        conan create ${{github.workspace}}/sopt --build missing -s compiler.cppstd=17 -o dompi=${{matrix.mpi}} -o openmp=${{matrix.omp}}
+        export CMAKE_PREFIX_PATH=${{github.workspace}}/local:$CMAKE_PREFIX_PATH
+        #export CMAKE_PREFIX_PATH=${{github.workspace}}/Catch2/build/lib/cmake:$CMAKE_PREFIX_PATH
+        mkdir -p ${{github.workspace}}/sopt/build
+        cd ${{github.workspace}}/sopt/build
+        cmake .. --fresh -DCMAKE_INSTALL_PREFIX=${{github.workspace}}/local -Ddompi=${{matrix.mpi}} -Dopenmp=${{matrix.omp}}
+        make -j$(nproc --ignore 1) install
 
-    - name: Dependencies
-      run: conan install ${{github.workspace}} -of ${{github.workspace}}/build -s compiler.cppstd=17 --build missing -o docasa=off -o dompi=${{matrix.mpi}} -o openmp=${{matrix.omp}}
 
     - name: Install
       # Build your program with the given configuration
-      run: conan build ${{github.workspace}} -of ${{github.workspace}}/build -s compiler.cppstd=17 -o docasa=off -o dompi=${{matrix.mpi}} -o openmp=${{matrix.omp}}
+      run: |
+        export CMAKE_PREFIX_PATH=${{github.workspace}}/local:$CMAKE_PREFIX_PATH
+        #export CMAKE_PREFIX_PATH=${{github.workspace}}/Catch2/build/lib/cmake:$CMAKE_PREFIX_PATH
+        #export CMAKE_PREFIX_PATH=${{github.workspace}}/sopt/build:$CMAKE_PREFIX_PATH
+        mkdir -p ${{github.workspace}}/build
+        cd ${{github.workspace}}/build
+        cmake .. --fresh -DCMAKE_INSTALL_PREFIX=${{github.workspace}}/local -Ddocasa=OFF -Ddompi=${{matrix.mpi}} -Dopenmp=${{matrix.omp}}
+        make -j$(nproc --ignore 1) install
 
     - name: Test
       working-directory: ${{github.workspace}}/build
       # Execute tests defined by the CMake configuration.
       # See https://cmake.org/cmake/help/latest/manual/ctest.1.html for more detail
-      run: ctest -C ${{env.BUILD_TYPE}} --output-on-failure
+      run: |
+        export LD_LIBRARY_PATH=${{github.workspace}}/local/lib:${{github.workspace}}/local/external/lib:${LD_LIBRARY_PATH}
```

Comment:
What's in `${{github.workspace}}/local/lib` and `${{github.workspace}}/local/external/lib`?

Reply:
The manually installed dependencies